src/encoded/schemas/death_circumstances.json

```diff
@@ -85,7 +86,7 @@
         },
         "cause_of_death_immediate_interval": {
             "title": "Cause Of Death Immediate Interval",
-            "description": "Interval of time from immediate cause of death to death in minutes",
+            "description": "Interval of time from immediate cause of death to death (minutes)",
```

Comment:
In the spreadsheet the TPC commented they would prefer units of "hours" for cause_of_death_immediate_interval, death_pronounced_interval, and ventilator_time.